### DISCUSSION TITLE
/trust type permissions not applying

### DISCUSSION THREAD

**Discussion by MrBudha808** [GriefPrevention/GriefPrevention#1523] [Q&A]
Only owners and operators are able to assign /trust and trust related permissions to other players. Is there another way for users to trust other users?

**Comment by Jikoo** [comment]
Sounds like you're looking for `/permissiontrust`, which allows claim owners to specify players who can trust other players up to their own trust level. I.e. if a user only has container trust and not build trust, they can only trust others up to container level.
